libpromises/monitoring_read.c

```diff
@@ -164,7 +164,19 @@ void Nova_LoadSlots(void)
     {
         if (i < ob_spare)
         {
-            fscanf(f, "%*[^\n]\n");
+            int r;
+            do
+            {
+                r = fgetc(f);
+                if (r == EOF)
+                {
+                    break;
+                }
+            } while (r != (int)'\n');
```

Comment:
Please move change the `if-break` into a second condition in the `do-while`.

Reply:
Thanks, fixed.